packages/config/src/bridges/aptos.ts

```diff
@@ -166,24 +148,34 @@ export const aptos: Bridge = {
     {
       accounts: [
         {
-          address: '0x902F09715B6303d4173037652FA7377e5b98089E',
-          type: 'Contract',
+          address: '0x971016EF5Bd9C71fA4ff34D731974d03cEFf5F05',
+          type: 'MultiSig',
         },
       ],
-      name: 'Aptos Bridge Relayer',
+      name: 'Aptos Multisig',
       description:
-        'Contract authorized to relay messages and - as a result - withdraw funds from the bridge.',
+        'Bridge owner, can setup tokens, fees, WETH token address (potentially malicious). Can withdraw all the funds from the Escrow after unlocking emergency withdrawal with 1 week delay.',
```

Comment:
Here we could mention the possibility of instantly changing config of L0 and spoofing withdrawals...